CHANGELOG.md

```diff
@@ -4,45 +4,45 @@
 
 ### Enhancements
 
-* Add support for `menuinst` v2. This version enables shortcuts in all platforms (Windows, Linux, macOS) through a new JSON schema (see [CEP-11](https://github.com/conda-incubator/ceps/blob/main/cep-11.md)). Old v1-style JSON menus are still supported. (#11882)
-* Stop using `requests`/`pip` vendored `chardet` package. Instead, explicitly depend on `charset_normalizer`. (#13171)
-* Add a new plugin hook, `CondaHealthCheck`, to be run as part of `conda doctor`. (#13186)
-* Add `activate` and `deactivate` to `--help` command list. (#13191)
-* Download largest packages first, preventing smaller packages from having to wait for larger packages. (#13248)
-* Show the used solver in `conda info` output to simplify debugging. (#13265)
+* Add support for `menuinst` v2, enabling shortcuts across all platforms (Windows, Linux, macOS) using a new JSON schema (see [CEP-11](https://github.com/conda-incubator/ceps/blob/main/cep-11.md)). Retain support for old v1-style JSON menus. (#11882)
+* Stop using vendored `chardet` package by `requests`/`pip`; explicitly depend on `charset_normalizer`. (#13171)
+* Introduce a new plugin hook, `CondaHealthCheck`, as part of `conda doctor`. (#13186)
+* Include `activate` and `deactivate` in the `--help` command list. (#13191)
+* Prioritize download of larger packages to prevent smaller ones from waiting. (#13248)
+* Display the used solver in `conda info` output for debugging purposes. (#13265)
 * Add `__conda` virtual package. (#13266)
 * Switch from `appdirs` to `platformdirs`. (#13306)
-* Resume interrupted package downloads. (#8695)
+* Implement resume capability for interrupted package downloads. (#8695)
 
 ### Bug fixes
 
-* Log expected JLAP range-request errors, which can happen when the remote file has rolled over, at `info` level. (#12913)
-* Fix a bug where not providing a command but providing an option like `--debug` results in an error. (#13232)
-* Improve CTRL-C (cancellation) handling of parallel download threads. (#13234)
-* Allow `CONDA_FETCH_THREADS`/`fetch_threads`, which sets parallel package downloads, to be overridden from its default value of `5`. (#13263)
-* Require `requests >=2.28` for improved `response.json()` exception handling. (#13346)
-* Use `callback=reset_context` in `conda.plan` to fix several `conda-build` + `conda-libmamba-solver` incompatibilities. ([conda-libmamba-solver#393](https://github.com/conda/conda-libmamba-solver/issues/393) and [conda-libmamba-solver#386](https://github.com/conda/conda-libmamba-solver/issues/386) via #13357)
+* Log expected JLAP range-request errors at `info` level, occurring when the remote file has rolled over. (#12913)
+* Fix a bug causing an error when options like `--debug` are used without specifying a command. (#13232)
+* Improve CTRL-C (cancellation) handling for parallel download threads. (#13234)
+* Allow overriding of `CONDA_FETCH_THREADS`/`fetch_threads` to set parallel package downloads beyond the default `5`. (#13263)
```

Comment:
A user would be more likely to want to decrease the number of parallel downloads.